<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deletedialog</class>
 <widget class="QDialog" name="deletedialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>286</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>572</width>
    <height>286</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>572</width>
    <height>286</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>4</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>应变传感器</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>151</x>
     <y>4</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>挠度传感器</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>4</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>位移传感器</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>4</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>561</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>应变传感器</string>
   </property>
   <widget class="QRadioButton" name="yb1radioButton">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>37</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>一号监测点</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="yb3radioButton">
    <property name="geometry">
     <rect>
      <x>233</x>
      <y>37</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>三号监测点</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="yb4radioButton">
    <property name="geometry">
     <rect>
      <x>344</x>
      <y>37</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>四号监测点</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="yb5radioButton">
    <property name="geometry">
     <rect>
      <x>455</x>
      <y>37</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>五号监测点</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="yb2radioButton">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>37</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>二号监测点</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="yb6radioButton">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>72</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>六号监测点</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="yb7radioButton">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>72</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>七号监测点</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="yb8radioButton">
    <property name="geometry">
     <rect>
      <x>233</x>
      <y>72</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>八号监测点</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="yb9radioButton">
    <property name="geometry">
     <rect>
      <x>344</x>
      <y>72</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>九号监测点</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="yb10radioButton">
    <property name="geometry">
     <rect>
      <x>455</x>
      <y>72</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>十号监测点</string>
    </property>
   </widget>
   <widget class="QPushButton" name="yb1pushButton">
    <property name="geometry">
     <rect>
      <x>455</x>
      <y>100</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="yb2pushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>删除全部</string>
    </property>
   </widget>
   <widget class="QPushButton" name="yb3pushButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>160</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>561</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>挠度传感器</string>
   </property>
   <widget class="QRadioButton" name="nd1radioButton">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>21</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>一号监测点</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="nd3radioButton">
    <property name="geometry">
     <rect>
      <x>227</x>
      <y>21</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>三号监测点</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="nd2radioButton">
    <property name="geometry">
     <rect>
      <x>114</x>
      <y>21</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>二号监测点</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="nd4radioButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>21</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>四号监测点</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="nd5radioButton">
    <property name="geometry">
     <rect>
      <x>453</x>
      <y>21</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>五号监测点</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="nd6radioButton">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>41</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>六号监测点</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="nd7radioButton">
    <property name="geometry">
     <rect>
      <x>114</x>
      <y>41</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>七号监测点</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="nd8radioButton">
    <property name="geometry">
     <rect>
      <x>227</x>
      <y>41</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>八号监测点</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="nd9radioButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>41</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>九号监测点</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="nd10radioButton">
    <property name="geometry">
     <rect>
      <x>453</x>
      <y>41</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>十号监测点</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="nd11radioButton">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>61</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>十一号监测点</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="nd12radioButton">
    <property name="geometry">
     <rect>
      <x>114</x>
      <y>60</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>十二号监测点</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="nd13radioButton">
    <property name="geometry">
     <rect>
      <x>227</x>
      <y>61</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>十三号监测点</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="nd14radioButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>61</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>十四号监测点</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nd1pushButton">
    <property name="geometry">
     <rect>
      <x>453</x>
      <y>60</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nd2pushButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>删除全部</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nd3pushButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>120</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>561</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>位移传感器</string>
   </property>
   <widget class="QRadioButton" name="wy1radioButton">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>34</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>一号监测点</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="wy2radioButton">
    <property name="geometry">
     <rect>
      <x>216</x>
      <y>34</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>二号监测点</string>
    </property>
   </widget>
   <widget class="QPushButton" name="wy1pushButton">
    <property name="geometry">
     <rect>
      <x>371</x>
      <y>31</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="wy2pushButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>删除全部</string>
    </property>
   </widget>
   <widget class="QPushButton" name="wy3pushButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
